devops/db_sync/code/requirements.txt

```diff
@@ -0,0 +1,5 @@
+system_logger_tool>=0.0.3
+system-config-tool>=0.0.3
+# bitarray>=2.8.1
+# pandas>=1.3.5
```

Comment:
same

Reply:
Done but this will not be used because main code of db_sync will be used instead